packages/framework/src/Resources/config/services.yml

```diff
@@ -8,6 +8,7 @@ parameters:
     faker.seed: 1234
     shopsys.entity_extension.map: {}
     shopsys.admin_locale: en
+    build-version: "0000000000000000"
```

Comment:
I think these default values are better here because we don't need to force users to make unnecessary changes in their parameters file as `parameters_common.yml` is a part of project-base (open-box)